raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -285,11 +180,17 @@ def test_matrix_message_sync(matrix_transports):
         canonical_identifier=factories.UNIT_CANONICAL_ID,
     )
 
+    raiden0_queues = views.get_all_messagequeues(views.state_from_raiden(raiden_service0))
+    raiden0_queues[queue_identifier] = []
+
     for i in range(5):
         message = Processed(message_identifier=i, signature=EMPTY_SIGNATURE)
+        # XXX: Fugly, this is a bad hack and shall not stand
```

Comment:
Should this be fixed in this PR?

Reply:
Ah that actually is the fixed version. I just forgot to remove the comment :)

This still could use improvement since it's easy to forget to manually handle the queues.